kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/internal/KotestEngineProperties.kt

```diff
@@ -52,6 +52,8 @@ object KotestEngineProperties {
 
    const val projectTimeout = "kotest.framework.projectTimeout"
 
+   const val logLevel = "kotest.framework.logLevel"
```

Comment:
prefer no camel case on this.
And just noticed projectTimeout could change too.